lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
It can't take `valid` because that's not a valid keyword anymore. I could write `int, iterable of int` etc. and then note in the text that it must be zero, would that be better?
I'm not so sure about including the default everywhere either, Martin started that. I can remove it again.